wuvt/static/js/admin/donation_index.js

```diff
@@ -4,4 +4,8 @@ $('#donations').DataTable({
     'order': [[3, 'desc']],
 });
 
+function confirm_delete(e){
+    if(!confirm('Are you sure you want to reset donation stats?'))e.preventDefault();
```

Comment:
Could you please split this across multiple lines so this is a bit more clear? Something like
```if(!confirm...) {
    e.preventDefault();
}```